en/debug/porting-guide.md

```diff
@@ -2,55 +2,67 @@
-  * FMUv5 example: [nuttx-configs/px4fmu-v5/include/board.h](https://github.com/PX4/Firmware/blob/master/nuttx-configs/px4fmu-v5/include/board.h).
-* NuttX OS config (created with menuconfig): [nuttx-configs/px4fmu-v5/nsh/defconfig](https://github.com/PX4/Firmware/blob/master/nuttx-configs/px4fmu-v5/nsh/defconfig).
-* Linker scripts and other required settings: [nuttx-configs](https://github.com/PX4/Firmware/tree/master/nuttx-configs). 
-  * FMUv5 example: [nuttx-configs/px4fmu-v5](https://github.com/PX4/Firmware/tree/master/nuttx-configs/px4fmu-v5).
-* Boot file system (startup script): [ROMFS/px4fmu\_common](https://github.com/PX4/Firmware/tree/master/ROMFS/px4fmu_common)
-* Board build configuration: [cmake/configs/nuttx\_px4fmu-v5\_default.cmake](https://github.com/PX4/Firmware/blob/master/cmake/configs/nuttx_px4fmu-v5_default.cmake).
-* Drivers: [src/drivers](https://github.com/PX4/Firmware/tree/master/src/drivers).
-* Reference config: Running `make px4fmu-v4_default` builds the FMUv4 config, which is the current NuttX reference configuration.
+## Host Operating System Configuration
 
+This section describes the purpose and location of the configuration files required for each supported host operating system to port them to new flight controller hardware.
 
-### NuttX Menuconfig
+### NuttX
 
-If you need to modify the NuttX [menuconfig](https://bitbucket.org/nuttx/nuttx) you can do this using the PX4 shortcuts:
-```sh
-make px4fmu-v2_default menuconfig
-make px4fmu-v2_default qconfig
-```
+In order to port PX4 on NuttX to a new hardware target, that hardware target must be supported by NuttX.  The NuttX project maintains an excellent [porting guide](http://www.nuttx.org/Documentation/NuttxPortingGuide.html) for porting NuttX to a new computing platform.
 
-## QuRT / Hexagon
+For all NuttX based flight controllers (e.g. the Pixhawk series) the OS is loaded as part of the application build.
 
-* The start script is located in [posix-configs/](https://github.com/PX4/Firmware/tree/master/posix-configs).
-* The OS configuration is part of the default Linux image (TODO: Provide location of LINUX IMAGE and flash instructions).
-* The PX4 middleware configuration is located in [src/drivers/boards](https://github.com/PX4/Firmware/tree/master/src/drivers/boards). TODO: ADD BUS CONFIG
-* Drivers: [DriverFramework](https://github.com/px4/DriverFramework).
-* Reference config: Running `make qurt_eagle_release` builds the Snapdragon Flight reference config.
+The configuration files for NuttX based boards, including linker scripts and other required settings are located under [platforms/nuttx/nuttx-configs](https://github.com/PX4/Firmware/tree/master/platforms/nuttx/nuttx-configs).
+
+The following example uses FMUv5 as it is the current [reference configuration](./reference-design.md) for NuttX based flight controllers:
```

Comment:
Calling FMUv5 the current reference config for NuttX based flight controllers feels a little odd because historically the variants are not necessarily sequential - ie FMU versions only really infer binary compatibility and a lower number could be the same level of advanced. I guess it is the most recent and the one we would hope manufacturers would target so leave as is.

Reply:
Understood about the binary compatibility/feature set, I had read that in other places as well, I had intended it only as you mentioned- If someone is porting to a brand new board, FMUv5 would be the best place to start.  I thought I had seen a similar sentence in the reference-design.md doc, but now I'm not... I will add a note here to remind people that the versions/generations to not correlate with capabilities, simply that v5, being the most recent version, reflects the most up to date methods for hardware and code structure.

Reply:
I actually wouldn't add a note here (maybe back in the reference-design doc though!). 
What I'd do is change as shown:

FROM
> The following example uses FMUv5 as it is the current [reference configuration](./reference-design.md) for NuttX based flight controllers:

TO

> The following example uses FMUv5 as it is a recent [reference configuration](./reference-design.md) for NuttX based flight controllers: